### PR TITLE
Dataviews postcard GA release

### PR DESCRIPTION
**Pull request by d-ng**
Updated documentation for GA release of data views with assets, summaries and Uoms

### CODE REVIEW

File: Content_Portal/Documentation/DataViews/DefineaDataView/Queries.md [hunk 1]
```diff
@@ -56,19 +56,18 @@ In the example scenario, each power inverter is associated with two streams. It
 
 These streams will be available in one single field set. This is good, because our data view can operate in terms of physical assets, and not be bound by how the stream data was recorded.
 
-## Asset queries (Coming Soon)
-*Note: This section covers features that are not yet generally available. If you are interested in trialing these pre-release features, contact your account team for more details.*
+## Asset queries
 
-Currently, the only supported query kind is Stream meaning only SDS streams can be queried and included in a data view as data items. With the introduction of asset queries, data views will also be able to query and include [assets](xref:AssetsProperties).
+If assets are defined with stream references, the data view can be created by directly querying for the assets.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Queries.md:61]: Change to: "Users can create a data view by querying directly for assets, as long as the assets have stream references defined."

File: Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ You define multiple aspects of a data view when you define it, including data it
 A data view must have a unique identifier. It may have a friendly name and description. If a friendly name is not specified, the identifier will be used as the data view's name.
 
 ### Include data items
-One or more queries determine the set of data items (such as SDS streams) that the data view will include. Each [Query](xref:DataViewsQueries) should represent a collection of like data items. To include data items that represent very different items, such as power inverters and weather stations, use separate queries.
+One or more queries determine the set of data items (such as streams and assets) that the data view will include. Each [Query](xref:DataViewsQueries) should represent a collection of like data items. To include data items that represent very different items, such as power inverters and weather stations, use separate queries.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md:17]: Last sentence works better as, "Use separate queries to include data items that represent very different items, such as power inverters and weather stations."

File: Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md [hunk 3]
```diff
@@ -92,18 +92,18 @@ A set of fields included in the data view, sharing a common role and query. One
 | IdentifyingField | Field | Optional | null | A field by which to tell the data items apart, within each group. Any field from field sources `FieldSource.Id`, `FieldSource.Name`, `FieldSource.Metadata` and `FieldSource.Tags` can be used as an identifying field.
 
 ### Field
-Individual piece of information, such as a property of an SDS stream or metadata of that stream.
+Individual piece of information, such as a property of an SDS stream, an asset stream reference, or metadata of that stream or asset.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md:95]: Add "An" in front: "An individual piece of information...."

File: Content_Portal/Documentation/DataViews/DefineaDataView/Rec_Workflow.md [hunk 4]
```diff
@@ -5,10 +5,10 @@ uid: DefineaDataViewRecommendedWorkflow
 # Recommended workflow: Define a data view
 This is an introduction to the recommended workflow for defining data views. The following sections show these concepts in action.
 
-Designing and tweaking a data view is expected to be an iterative process. You are not expected to "get it right the first time," nor to be an expert at your OCS configuration such as the streams available in the Sequential Data Store. Data views provide an interactive way to select, identify, and arrange the data you and your applications need.
+Designing and tweaking a data view is expected to be an iterative process. You are not expected to "get it right the first time," nor to be an expert at your OCS configuration such as the streams and assets available. Data views provide an interactive way to select, identify, and arrange the data you and your applications need.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Rec_Workflow.md:8]: Change to, "...OCS configuration such as available streams and assets."

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md [hunk 5]
```diff
@@ -2,25 +2,22 @@
...
-
-*Note: This topic covers features that are not yet generally available. If you are interested in trialing these pre-release features, contact your account team for more details.*
+# Display calculated summaries
 
 Data views can be configured to show summary calculations for streams and assets.  Summary calculations are defined as the summary types enumerated in [SdsSummaryType](xref:sdsReadingDataApi#get-summaries). This topic describes how to include summaries in a data view. 
 
-# Available summaries
-Not all SDS properties will have summaries. For example a string property has no Mean value. If a data item field requests a summary that is unavailable these values will be null in the view data.  
+## Available summaries
+Not all properties have summaries. For example a string property has no Mean value. If a data item field requests a summary that is unavailable these values will be null in the view data.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md:10]: For example, (add comma)

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md [hunk 6]
```diff
@@ -2,25 +2,22 @@
...
 
 Data views can be configured to show summary calculations for streams and assets.  Summary calculations are defined as the summary types enumerated in [SdsSummaryType](xref:sdsReadingDataApi#get-summaries). This topic describes how to include summaries in a data view. 
 
-# Available summaries
-Not all SDS properties will have summaries. For example a string property has no Mean value. If a data item field requests a summary that is unavailable these values will be null in the view data.  
+## Available summaries
+Not all properties have summaries. For example a string property has no Mean value. If a data item field requests a summary that is unavailable these values will be null in the view data.  
 
-# Including summaries in a dataview
+## Including summaries in a dataview
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md:12]: No gerunds in headings: Change to "Include summaries in a data view"

File: Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md [hunk 7]
```diff
@@ -32,8 +32,8 @@ The data view owner and access control list control basic authorization to a giv
 | Update access control | `ManageAccessControl` |
 
 ## Data access control
-The access control rights of each data item, such as a stream (see [Streams](xref:sdsStreams)), apply globally within OSIsoft Cloud Services, including within data views. When a data view [resolves](xref:ResolvedDataView) per-user, each [`Query`](xref:DataViewsQueries) is executed with the user's own permissions, honoring the access control configured for each data item. Read access to a data view does not affect the user's read access to (or even the visibility of!) any streams or other data items.
+The access control rights of each data item, such as a [stream](xref:sdsStreams) or [asset](xref:AssetsProperties), apply globally within OSIsoft Cloud Services, including within data views. When a data view [resolves](xref:ResolvedDataView) per-user, each [`Query`](xref:DataViewsQueries) is executed with the user's own permissions, honoring the access control configured for each data item. Read access to a data view does not affect the user's read access to (or even the visibility of!) any streams or assets.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md:35]: Delete exclamation point from (or even the visibility of) - - visually confusing

File: Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md [hunk 8]
```diff
@@ -32,8 +32,8 @@ The data view owner and access control list control basic authorization to a giv
...
 
 When you inspect the _data items_ that resolve for a data view, you see only the data items that you have `Read` access to.
 
-Configuring access control for streams and other data items is similar to data views. See the [Role-based access control](xref:accessControl) and [Streams](xref:sdsStreams) sections for more information.
+Configuring access control for streams and assets is similar to data views. See the [Role-based access control](xref:accessControl), [streams](xref:sdsStreams), and [assets](xref:AssetOrAssetTypeAccessControlAPI) sections for more information.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md:39]: Change to, "Configuring access control for streams and assets is similar to configuring data views."

**Review by osimarkrogers — CHANGES_REQUESTED** [review]

File: Content_Portal/Documentation/DataViews/Data_Views_Overview.md [hunk 9]
```diff
@@ -18,7 +18,7 @@ The ability to create data views in OCS will mesh directly with OSIsoft's Data S
 
 Understanding how the data view resolves into content is important both for the designer and the consumer. Data views provide extensive visibility into what data is included. The concept of "resolving" a data view is when the data view's queries and logic are executed to determine the data view's exact schema and content.
 
-The collection(s) of OCS resources (such as streams) that a data view acts on is the result of one or more queries. Streams and other OCS resources are collectively known as *data items*. Values from the data items' properties can be included in the data view as *fields*, as well as information about the data items (such as Id and Metadata).
+The collection(s) of OCS resources (such as streams and assets) that a data view acts on is the result of one or more queries. Streams, assets and other OCS resources are collectively known as *data items*. Values from the data items' properties can be included in the data view as *fields*, as well as information about the data items (such as Id and Metadata).
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/Data_Views_Overview.md:21]: fix in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md [hunk 10]
```diff
@@ -126,6 +126,76 @@ The result is much more consumable. The data field identifiers are no longer amb
 | - | Primary | ..Primary..PwrIn/PropertyId:Value | ..Primary..PwrIn/Tags |  ..Primary..PwrOut/PropertyId:Value | ..Primary..PwrOut/Tags  |
 | - | Secondary | ..Secondary..PwrIn/PropertyId:Value | ..Secondary..PwrIn/Tags |  ..Secondary..PwrOut/PropertyId:Value | ..Secondary..PwrOut/Tags  |
 
+## Working with assets
+
+If assets are defined, data field sets can be defined to show the properties from stream references of the assets. 
+
+To define data field sets with asset properties, two pieces of information are required: the stream reference name and the property id. In order to reference an asset property, a field must contain at least one stream reference name in the `.StreamReferenceNames` collection and at least one property id in the `.Keys` collection.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md:133]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md [hunk 11]
```diff
@@ -126,6 +126,76 @@ The result is much more consumable. The data field identifiers are no longer amb
...
+
+`.StreamReferenceNames` only applies to fields with source `FieldSource.PropertyId`. For all other field sources, `.StreamReferenceNames` are ignored. 
+
+Similar to `.Keys`, `.StreamReferenceNames` are evaluated in order specified until a match is found, i.e. first-match-wins.
+
+### Example: Defining data field sets with assets
+
+Let us take a subset of the power inverter streams in the [asset examples](xref:DataViewsAssetExamples), returned by the `Query` [value](xref:AssetsSearchAPI) `"Id:*Inverter* AND Metadata/Value:Winterthur"`.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md:141]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md [hunk 12]
```diff
@@ -126,6 +126,76 @@ The result is much more consumable. The data field identifiers are no longer amb
...
+```
+
+The view resolves nicely into the following results:
+
+| Timestamp | Meter | Power In Value | Power Out Value |
+|--|--|--|--|
+| - | Primary | ..Primary..PwrIn/PropertyId:Value | ..Primary..PwrOut/PropertyId:Value |
+| - | Secondary | ..Secondary..PwrIn/PropertyId:Value | ..Secondary..PwrOut/PropertyId:Value |
+
+The presence of context allow data view to align the data fields, without the addition of an `.IdentifyingField` as shown in the stream example above.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Field_Sets.md:197]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Queries.md (hunk 1 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Queries.md:61]: Fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md (hunk 2 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md:17]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md (hunk 3 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md:95]: Fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Rec_Workflow.md (hunk 4 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Rec_Workflow.md:8]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md (hunk 5 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md:10]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md (hunk 6 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Summaries.md:12]: fixed in next update

File: Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md (hunk 7 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md:35]: fixed in next update

File: Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md (hunk 8 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/SecureDataViews/Securing_Data_Views.md:39]: fixed in next update

**Comment by d-ng** [comment]
@osimarkrogers  changes committed and pushed. thank you!

**Review by osimarkrogers — APPROVED** [review]

**Pull request merged**